src/bearhug.js

```diff
@@ -105,9 +105,9 @@ angular.module('talis.bearhug', [])
     var oauthTokenRetryFail = false;
 
     // intercept for oauth tokens
-    $httpProvider.responseInterceptors.push([
-        '$q', '$injector','$location','applicationLoggingService',
-        function ($q, $injector, $location, applicationLoggingService) {
+    $httpProvider.interceptors.push([
+        '$q', '$injector','$location',
```

Comment:
Can we use `ngInject` or otherwise drop the strings-plus-function-variables DI convention?